media_service/mediastore.py

```diff
@@ -24,6 +27,30 @@
 class MediaStoreUploadException(Exception):
     pass
 
+def process_zip(filelist):
```

Comment:
Maybe this should be renamed to `process_file_uploads()` since it doesn't *just* deal with ZIP files like the name implies?